resources/public/api/conf/3.0/schemas/retrieve/calculation/foreignIncome.json

```
@@ -1,13 +1,146 @@
+            "type": "number",
+            "example": "501.99"
+          },
+          "netIncome": {
+            "description": "Net income for overseas pensions, state benefits and royalties. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+            "type": "number",
+            "example": "502.99"
+          },
+          "taxDeducted": {
+            "description": "Tax deducted for overseas pensions, state benefits and royalties. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+            "type": "number",
+            "example": "503.99"
+          },
+          "foreignTaxCreditRelief": {
+            "description": "A boolean indicating whether it's a Foreign Tax Credit Relief. This value must be true or false.",
+            "type": "boolean",
+            "example": "true"
+          }
+        },
+        "required": [
+          "countryCode"
+        ],
+        "additionalProperties": false
+      }
+    },
+    "chargeableAllOtherIncomeReceivedWhilstAbroad": {
+      "description": "Chargeable income received whilst abroad. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+      "type": "number",
+      "example": "504.99"
+    },
+    "allOtherIncomeReceivedWhilstAbroadDetail": {
+      "description": "Array detailing all other income received whilst abroad.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "countryCode": {
+            "description": "A three-letter code that represents a country name. The value must in ISO 3166-1 Alpha-3 format.",
+            "type": "string",
+            "example": "FRA"
+          },
+          "grossIncome": {
+            "description": "Gross income from all other income received whilst abroad. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+            "type": "number",
+            "example": "502.99"
+          },
+          "netIncome": {
+            "description": "Net income from all other income received whilst abroad. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+            "type": "number",
+            "example": "503.99"
+          },
+          "taxDeducted": {
+            "description": "Tax deducted from all other income received whilst abroad. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+            "type": "number",
+            "example": "504.99"
+          },
+          "foreignTaxCreditRelief": {
+            "description": "A boolean indicating whether it's a Foreign Tax Credit Relief. The value must be true or false.",
+            "type": "boolean",
+            "example": "true"
+          }
+        },
+        "required": [
+          "countryCode"
+        ],
+        "additionalProperties": false
+      }
+    },
+    "overseasIncomeAndGains": {
+      "description": "Array detailing overseas income and gains.",
```

suspect this should say:
```suggestion
      "description": "Object detailing overseas income and gains.",
```
as it's an object in our spec and downstream specs.